MAINTAINERS.yml

```diff
@@ -0,0 +1,1050 @@
+    # Provisional entry, to be corrected
+    # TODO: The entries in the maintainers spreadsheet do not match CODEWONERS at all!
+    status: maintained
+    maintainers:
+        - ruuddw
+    collaborators:
+        - vonhust
+    files:
+        - arch/nios2/
+        - include/arch/nios2/
+    labels:
+        - "area: NIOS2"
+
+"nRF52 BSIM":
+    status: maintained
+    maintainers:
+        - aescolar
+    files:
+        - boards/posix/nrf52_bsim/
+    labels:
+        - "platform: nrf52_bsim"
+
+"POSIX API layer":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - pfalcon
+    files:
+        - include/posix/
+        - lib/posix/
+        - tests/posix/
+    labels:
+        - "area: POSIX"
+
+"Power management":
+    # TSC : Note this entry differs from the spreadsheet:
+    # pizi-nordic is not anymore working in Zephyr, therefore he cannot be a
+    # maintainer, and the are is orphaned until a new maintainer is appointed
+    status: orphaned
+    collaborators:
+        - wentongwu
+    files:
+        - include/power/power.h
+        - samples/subsys/power/
+        - subsys/power/
+    labels:
+        - "area: Power Management"
+
+"RISCV arch":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nategraff-sifive
+    collaborators:
+        - mgielda
+    files:
+        - arch/riscv/
+        - include/arch/riscv/
+    labels:
+        - "area: RISCV"
+
+Sanitycheck:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nashif
+    files:
+        - scripts/sanitycheck
+        - scripts/sanity_chk/
+    labels:
+        - "area: Sanitycheck"
+
+Settings:
+    status: maintained
+    maintainers:
+        - nvlsianpu
+    files:
+        - include/settings/
+        - subsys/settings/
+        - tests/subsys/settings/
+        - doc/reference/runtime_conf/
+    labels:
+        - "area: Settings"
+
+Shell:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - jakub-uC
+    files:
+        - include/shell/
+        - samples/subsys/shell/
+        - subsys/shell/
+        - tests/shell/
+        - tests/subsys/shell/
+    labels:
+        - "area: Shell"
+
+Shields:
+    status: maintained
+    maintainers:
+        - erwango
+    collaborators:
+        - avisconti
+        - jfischer-phytec-iot
+    files:
+        - boards/shields/
+    labels:
+        - "area: Shields"
+
+STM32:
+    status: maintained
+    maintainers:
+        - erwango
+    files:
+        - boards/arm/nucleo_*/
+        - boards/arm/stm32*_disco/
+        - boards/arm/stm32*_eval/
+        - drivers/*/*stm32*/
+        - drivers/*/*/*stm32*
+        - dts/arm/st/
+        - dts/bindings/*/*stm32*
+        - soc/arm/st_stm32/
+    labels:
+        - "platform: STM32"
+    description: >-
+        STM32 SOCs, dts files and related drivers. ST nucleo, disco and eval
+        boards.
+
```

Comment:
BOARDS(???):
    status: maintained
    maintainers:
        - avisconti
    files:
        - boards/arm/96b_argonkey/ 
        - boards/arm/sensortile_box/ 
    labels:
        - 
    description: >-
        ....

Reply:
@avisconti, I guess this has to be put in a boards dedicated section

Reply:
@erwango Yes, I guess it is still missing.